README.md

```diff
@@ -100,7 +100,7 @@ The installed `marp` command is available in [npm-scripts](https://docs.npmjs.co
 You can install with `-g` option if you want to use `marp` command globally.
 
 ```bash
-npm install -g @marp-team/marp-cli
+sudo npm install -g @marp-team/marp-cli
```

Comment:
**It's just evil.** `sudo` with `npm` command should not use, and Marp team do never want to recommend it.

- https://medium.com/@ExplosionPills/dont-use-sudo-with-npm-5711d2726aa3
- https://medium.com/@ExplosionPills/dont-use-sudo-with-npm-still-66e609f5f92
- https://givan.se/do-not-sudo-npm/

Reply:
Thank you. I understand the concerns regarding the use of `sudo` with `npm` commands. I will close this PR.